sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/TransactionContext.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import java.nio.ByteBuffer;
+
+public class TransactionContext {
```

Comment:
I would hold off on adding types we haven't discussed until they are fleshed out.